prow/cmd/deck/templates.go

```diff
@@ -88,6 +89,13 @@ func handleSimpleTemplate(o options, cfg config.Getter, templateName string, par
 			http.Error(w, "error parsing template", http.StatusInternalServerError)
 			return
 		}
+
+		// ensures that the CSRF token is always passed to the template. If CSRF protection is not
+		// set up, the token will be an empty string.
+		if param == nil {
+			param = struct{ CSRFToken string }{csrf.Token(r)}
```

Comment:
I don't really like this defaulting here - let's move this down into the base template as described above.